gtk4/treeview/treeview.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"github.com/diamondburned/gotk4/pkg/core/glib"
+	"github.com/diamondburned/gotk4/pkg/gtk/v4"
+	"os"
+)
+
+// IDs to access the tree view columns by
+const (
+	COLUMN_NAME = iota
+	COLUMN_COMMENT
+)
+
+// ItemList is a aimple wrapper around gtk.ListStore
+type ItemList struct {
+	*gtk.ListStore
+}
+
+// NewItemList create a new list of items
+func NewItemList() *ItemList {
+	listStore := gtk.NewListStore([]glib.Type{glib.TypeString, glib.TypeString})
+	return &ItemList{listStore}
+}
+
+// Add adds a new item to the list
+func (i *ItemList) Add(name, comment string) {
+	iter := i.Append()
+
+	// FIXME: doesn't seem to work
+	//listStore.Set(&iter,
+	//	[]int{COLUMN_NAME, COLUMN_COMMENT},
+	//	[]glib.Value{*glib.NewValue(name), *glib.NewValue(comment)})
```

Comment:
@diamondburned any idea what's going on here? the value of a column is missing if I use this.

Reply:
It seems like the dereference here may cause `glib.Value` to be freed earlier
than it should be, because that throws away the Finalizer. This would be a bug
in `pkg/core/glib`, I think.